pkg/framework/capi-machines.go

```diff
@@ -0,0 +1,56 @@
+package framework
```

Comment:
The idiomatic way of naming multi word filenames is with `_` (source: https://www.mohitkhare.com/blog/go-naming-conventions/)

Reply:
Thanks